src/c-writer.cc

```diff
@@ -601,24 +601,20 @@ void CWriter::DefineImportName(const Import* import,
   import_syms_.insert(name);
   import_module_sym_map_.emplace(name, import->module_name);
   global_syms_.insert(mangled);
-  bool success =
-      global_sym_map_.emplace(name + MangleField(type), mangled).second;
-  assert(success);
+  assert(global_sym_map_.emplace(name + MangleField(type), mangled).second);
```

Comment:
Sadly you can't do this because `global_sym_map_.emplace` has a side effect that we want..

Reply:
You can do this but its a bit ugly:

```
#ifndef NDEBUG
   bool success =
#endif
global_sym_map_.emplace(name + MangleField(type), mangled).second;
```

Reply:
Actually answer from here looks like the nicest way: https://stackoverflow.com/questions/777261/avoiding-unused-variables-warnings-when-using-assert-in-a-release-build